weather_mv/loader_pipeline/pipeline.py

```diff
@@ -75,7 +76,11 @@ def pipeline(known_args: argparse.Namespace, pipeline_args: t.List[str]) -> None
         elif known_args.subcommand == 'regrid' or known_args.subcommand == 'rg':
             paths | "Regrid" >> Regrid.from_kwargs(**vars(known_args))
         elif known_args.subcommand == 'earthengine' or known_args.subcommand == 'ee':
-            paths | "MoveToEarthEngine" >> ToEarthEngine.from_kwargs(**vars(known_args))
+            pipeline_options = PipelineOptions(pipeline_args)
+            pipeline_options_dict = pipeline_options.get_all_options()
```

Comment:
Please add a comment, that we modified this because we want to access `project` etc in later stage of the pipeline.